context/package.json

```
@@ -48,7 +48,7 @@
     "uuid": "^8.3.2",
     "vega": "^5.13.0",
     "vega-lite": "^4.13.1",
-    "vitessce": "^1.1.13",
+    "vitessce": "1.1.14",
```

```suggestion
    "vitessce": "^1.1.14",
```
I'd keep the `^`: It's the normal practice with NPM. Without it I'm wondering if there's something wrong with a higher version.